diff --git a/node_modules/gatsby-remark-autolink-headers/gatsby-browser.js b/node_modules/gatsby-remark-autolink-headers/gatsby-browser.js
deleted file mode 100644
index 2f80613..0000000
--- a/node_modules/gatsby-remark-autolink-headers/gatsby-browser.js
+++ /dev/null
@@ -1,33 +0,0 @@
-"use strict";
-
-var offsetY = 0;
-var getTargetOffset = function getTargetOffset(hash) {
-  var id = window.decodeURI(hash.replace("#", ""));
-  if (id !== "") {
-    var element = document.getElementById(id);
-    if (element) {
-      var scrollTop = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop;
-      var clientTop = document.documentElement.clientTop || document.body.clientTop || 0;
-      var computedStyles = window.getComputedStyle(element);
-      var scrollMarginTop = computedStyles.getPropertyValue("scroll-margin-top") || computedStyles.getPropertyValue("scroll-snap-margin-top") || "0px";
-      return element.getBoundingClientRect().top + scrollTop - parseInt(scrollMarginTop, 10) - clientTop - offsetY;
-    }
-  }
-  return null;
-};
-exports.onInitialClientRender = function (_, pluginOptions) {
-  if (pluginOptions.offsetY) {
-    offsetY = pluginOptions.offsetY;
-  }
-  requestAnimationFrame(function () {
-    var offset = getTargetOffset(window.location.hash);
-    if (offset !== null) {
-      window.scrollTo(0, offset);
-    }
-  });
-};
-exports.shouldUpdateScroll = function (_ref) {
-  var location = _ref.routerProps.location;
-  var offset = getTargetOffset(location.hash);
-  return offset !== null ? [0, offset] : true;
-};
\ No newline at end of file
diff --git a/node_modules/gatsby-remark-autolink-headers/gatsby-ssr.js b/node_modules/gatsby-remark-autolink-headers/gatsby-ssr.js
index df19ec0..8a29f0c 100644
--- a/node_modules/gatsby-remark-autolink-headers/gatsby-ssr.js
+++ b/node_modules/gatsby-remark-autolink-headers/gatsby-ssr.js
@@ -15,15 +15,9 @@ exports.onRenderBody = function (_ref, pluginOptions) {
     offsetY = _Object$assign.offsetY;
   var styles = "\n    ." + className + ".before {\n      position: absolute;\n      top: 0;\n      left: 0;\n      transform: translateX(-100%);\n      padding-right: 4px;\n    }\n    ." + className + ".after {\n      display: inline-block;\n      padding-left: 4px;\n    }\n    h1 ." + className + " svg,\n    h2 ." + className + " svg,\n    h3 ." + className + " svg,\n    h4 ." + className + " svg,\n    h5 ." + className + " svg,\n    h6 ." + className + " svg {\n      visibility: hidden;\n    }\n    h1:hover ." + className + " svg,\n    h2:hover ." + className + " svg,\n    h3:hover ." + className + " svg,\n    h4:hover ." + className + " svg,\n    h5:hover ." + className + " svg,\n    h6:hover ." + className + " svg,\n    h1 ." + className + ":focus svg,\n    h2 ." + className + ":focus svg,\n    h3 ." + className + ":focus svg,\n    h4 ." + className + ":focus svg,\n    h5 ." + className + ":focus svg,\n    h6 ." + className + ":focus svg {\n      visibility: visible;\n    }\n  ";
 
-  // This script used to have `let scrollTop` and `let clientTop` instead of
-  // current ones which are `var`. It is changed due to incompatibility with
-  // older browsers (that do not implement `let`). See:
-  //  - https://github.com/gatsbyjs/gatsby/issues/21058
-  //  - https://github.com/gatsbyjs/gatsby/pull/21083
-  var script = "\n    document.addEventListener(\"DOMContentLoaded\", function(event) {\n      var hash = window.decodeURI(location.hash.replace('#', ''))\n      if (hash !== '') {\n        var element = document.getElementById(hash)\n        if (element) {\n          var scrollTop = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop\n          var clientTop = document.documentElement.clientTop || document.body.clientTop || 0\n          var offset = element.getBoundingClientRect().top + scrollTop - clientTop\n          // Wait for the browser to finish rendering before scrolling.\n          setTimeout((function() {\n            window.scrollTo(0, offset - " + offsetY + ")\n          }), 0)\n        }\n      }\n    })\n  ";
+  var script = "\n";
   var style = icon ? /*#__PURE__*/_react.default.createElement("style", {
     key: "gatsby-remark-autolink-headers-style",
-    type: "text/css"
   }, styles) : undefined;
   return setHeadComponents([style, /*#__PURE__*/_react.default.createElement("script", {
     key: "gatsby-remark-autolink-headers-script",
